osi_sensorviewconfiguration.proto

```diff
@@ -752,27 +752,126 @@ message CameraSensorViewConfiguration
         //
         CHANNEL_FORMAT_RGB_F32_LIN = 9;
 
-        // Bayer RGGB Channels UINT8 FP Linear.
+        // Bayer BGGR Channels UINT8 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U8_LIN = 10;
 
-        // Bayer RGGB Channels UINT16 FP Linear.
+        // Bayer BGGR Channels UINT16 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U16_LIN = 11;
 
-        // Bayer RGGB Channels UINT32 FP Linear.
+        // Bayer BGGR Channels UINT32 FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_U32_LIN = 12;
 
-        // Bayer RGGB Channels Single Precision FP Linear.
+        // Bayer BGGR Channels Single Precision FP Linear.
         //
         CHANNEL_FORMAT_BAYER_BGGR_F32_LIN = 13;
 
+        // Bayer RGGB Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U8_LIN = 14;
+
+        // Bayer RGGB Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U16_LIN = 15;
+
+        // Bayer RGGB Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_U32_LIN = 16;
+
+        // Bayer RGGB Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_BAYER_RGGB_F32_LIN = 17;
+
+        // Read Clear Clear Clear Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U8_LIN = 18;
+
+        // Read Clear Clear Clear Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U16_LIN = 19;
+
+        // Read Clear Clear Clear Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_U32_LIN = 20;
+
+        // Read Clear Clear Clear Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_RCCC_F32_LIN = 21;
+
+        // Read Clear Clear Blue Channels UINT8 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U8_LIN = 22;
+
+        // Read Clear Clear Blue Channels UINT16 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U16_LIN = 23;
+
+        // Read Clear Clear Blue Channels UINT32 FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_U32_LIN = 24;
+
+        // Read Clear Clear Blue Channels Single Precision FP Linear.
+        //
+        CHANNEL_FORMAT_RCCB_F32_LIN = 25;
+        
+
         // TBD: Further channel permutations and padding (e.g. RGBZ,
         // BGR, BAYER_RGGB/GBRG/GRBG/...), non-BAYER filters, non-linear
         // encodings, ...
     }
 
+    // Number of samples per pixel.
```

Comment:
Maybe this part of the changes should be split into a new PR, to keep the PRs thematically more closely related? No strong feeling, just an idea, since reviewing this part needs a deeper look, I think, than the stuff above...

Reply:
Currently, all issues are addressed if new issues arise then we can look into splitting this PR.